city_scrapers/spiders/cook_housing.py

```diff
@@ -0,0 +1,91 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD, FORUM, NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class CookHousingSpider(CityScrapersSpider):
+    name = "cook_housing"
+    agency = "Cook County Housing Authority"
+    timezone = "America/Chicago"
+    allowed_domains = ["thehacc.org"]
+    start_urls = ["http://thehacc.org/events/"]
```

Comment:
Since you can get different months from the URL, we should try and get 2 months in the past, the current month and then 2 months in the future. Here's an example: https://github.com/City-Bureau/city-scrapers/blob/aa2caea9337d3a509d13302d4a0523c2654f39e6/city_scrapers/spiders/det_library_commission.py#L17-L25

Reply:
Thanks for the example!